packages/clerk-js/src/ui/Components.tsx

```diff
@@ -342,6 +352,20 @@ const Components = (props: ComponentsProps) => {
     };
 
     componentsControls.openModal = (name, props) => {
+      // Prevent opening enableOrganizations prompt if it's already open
```

Comment:
I've added this handling to avoid updating the state multiple times if there's already a `enableOrganizationsPrompt` entry.

Since this can be called multiple times if there are multiple components/hooks on the same page:

<img width="560" height="567" alt="CleanShot 2025-11-12 at 20 00 00" src="https://github.com/user-attachments/assets/f4126fe5-3fa4-44b3-a1eb-c8f81e137679" />

Reply:
Our components also use organizations hooks internally, and therefore prompting with it would cause developers to be confused: "i've rendered `OrganizationSwitcher` but the prompt is mentioning `useOrganization`"

So I believe it makes sense to open the prompt on the first call and avoid updating the state afterwards.